docker-compose.yml

```diff
@@ -8,6 +8,7 @@ services:
       - ENV=development
       - MEMCACHED_HOST=memcached
       - 'MONGO_URL=mongodb://mongo:27017/electricity'
+      - TOKEN='7466690c-c66a-4a00-8e21-2cb7d538f380'
```

Comment:
The point is to remove the token from the code, so it also needs to be removed from here.
The way to do it is to tell docker-compose to read the token from the environment, see https://docs.docker.com/compose/environment-variables/#/substituting-environment-variables-in-compose-files
So you need to write `- 'TOKEN=${ENTSOE_TOKEN}'`. When you run your docker-compose command you just need to make sure the ENTSOE_TOKEN environment variable is declared in your terminal